src/client/containers/SelectionPage/HYF.logo.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="59" height="50" viewBox="0 0 59 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="250" height="150" viewBox="0 0 59 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are you sure about this change? Changing SVGs manually might not do what you think it does.